enginetest/queries/variable_queries.go

```diff
@@ -334,7 +334,7 @@ var VariableQueries = []ScriptTest{
 		},
 		Query: "SELECT @myvar",
 		Expected: []sql.Row{
-			{123.4},
+			{"123.4"},
```

Comment:
Does MySQL treat this as a string, too? I'm surprised that having a decimal in the value would cause the user var to be treated as a string.

Reply:
ooooh... actually... I bet this is because it's a decimal and our tests turn decimal values into strings for easier comparison, right?

Reply:
yep! 🙂